desktop/desktop_panels.xml

```diff
@@ -431,6 +439,8 @@
 						local nCurrentRound = DB.getValue(CombatManager.CT_ROUND, 0);
 						nCurrentRound = nCurrentRound % 10
 						DB.setValue(CombatManager.CT_ROUND, 'number', nCurrentRound);
+
+						DB.setValue("calendar.addyear", "number", getValue() or 0);
 					end
 				</script>
 			</basicnumber>
```

Comment:
All of these panel changes do the same things... in the onInit() for the number control, it gets the value from the DB or uses zero if not found.  Then in the onClose() and onDoubleClick/Update() the value of the control is saved in the DB.